package.json

```diff
@@ -303,19 +331,34 @@
             },
             {
                 "command": "language-julia.clearAllInlineResultsInEditor",
-                "key": "ctrl+I ctrl+C",
+                "key": "ctrl+i ctrl+c",
                 "when": "editorTextFocus && editorLangId == julia"
             },
             {
                 "command": "language-julia.chooseModule",
-                "key": "Ctrl-J Ctrl+M",
+                "key": "ctrl+j ctrl+m",
                 "when": "editorTextFocus && editorLangId == julia"
             },
             {
                 "command": "language-julia.executeJuliaCodeInREPL",
-                "key": "ctrl+Enter",
+                "key": "ctrl+enter",
                 "when": "editorTextFocus && editorLangId == juliamarkdown"
             },
+            {
+                "command": "language-julia.show-documentation",
+                "key": "ctrl+j ctrl+d",
```

Comment:
Should be `Alt+J Alt+D` if we go with https://github.com/julia-vscode/julia-vscode/pull/1454.